lib/src/date-input/Calendar.tsx

```diff
@@ -153,38 +162,45 @@ const Calendar = ({
         break;
     }
   };
-
   return (
-    <CalendarContainer>
+    <CalendarContainer role="grid">
       <DxcFlex alignItems="center" justifyContent="space-between">
```

Comment:
I think that it would be very helpful to add a `role` prop to our current `DxcFlex` component to make it more comfortable to meet requirements.
In the current state of this PR, using `DxcFlex` limits us to not be able to use role="row" for the header column (the one that has the name of the weekdays).

Reply:
It has been decided that DxcFlex should not have a role prop, so another implementation was done instead.